Update users-prompted-credentials-after-adding-shared-mailbox.md (#1507)
MicrosoftDocs/OfficeDocs-Support: Exchange/ExchangeHybrid/user-and-shared-mailboxes/users-prompted-credentials-after-adding-shared-mailbox.md

To work around the problem, move the shared mailbox to the on-premises environment.

## Solution

> By default, Outlook tries to connect to the legacy public folders for the shared mailbox account. If public folders were created within the organization or if an Exchange Online organization is configured to access on-premises public folders, all clients would make a connection to and show the “Public folders” object in Outlook.

Reviewer: ```suggestion
By default, Outlook tries to connect to the legacy public folders for the shared mailbox account. If public folders were created within the organization or if an Exchange Online organization is configured to access on-premises public folders, all clients would make a connection to and show the public folders object in Outlook.
```

> We can modify  to show the public folder object in Outlook to only a set of users who might need them. To get started, this will be available for Outlook for Windows users only. To do this, administrators can use two parameters:

Reviewer: ```suggestion
However, this can be modified to show the public folders object in Outlook to only a set of users who might need them. Initially, this will be available for Outlook for Windows users only. To modify this default, administrators can use two parameters:
```

1. In a PowerShell window, load the EXO V2 module by running the following command:

   ```powershell
   Import-Module ExchangeOnlineManagement
   ```
   
2. The command that you need to run uses the following syntax:

   ```powershell
   Connect-ExchangeOnline -UserPrincipalName <UPN>
   ```

> 3. Enable  **PublicFolderClientAccess** on the user object. By default, its value is set to ‘false’. Setting this to ‘true’ on a specific user designates this user as one of users who will see public folders in Outlook. In this example. We are enabling access to only the sharedmailbox "Accounts".

Reviewer: ```suggestion
3. Enable  **PublicFolderClientAccess** on the user object. By default, its value is set to false. Setting this value to true on a specific user designates this user as one of the users who will see public folders in Outlook. In this example. We are enabling access to only the **Accounts** shared mailbox.
```

Author: I feel the False and true part can be as it is. But I have made some changes with the wordings on the shared mailbox per your review.

> 3. Enable  **PublicFolderClientAccess** on the user object. By default, its value is set to ‘false’. Setting this to ‘true’ on a specific user designates this user as one of users who will see public folders in Outlook. In this example. We are enabling access to only the **Accounts** shared mailbox..

Reviewer: ```suggestion
3. Enable  **PublicFolderClientAccess** on the user object. By default, its value is set to ‘false’. Setting this to ‘true’ on a specific user designates this user as one of the users who will see public folders in Outlook. In this example. We are enabling access to only the **Accounts** shared mailbox.
```

   ```powershell
   Set-CASMailbox accounts@contoso.com -PublicFolderClientAccess $true
   ```
   

> 4. Enable **PublicFolderShowClientControl** parameter on the organization config. By default, the value of this parameter is also ‘false’ and once it is set to ‘true’, the controlled access of public folders is enabled.

Reviewer: ```suggestion
4. Enable the **PublicFolderShowClientControl** parameter on the organization configuration. By default, the value of this parameter is also set to _false_ and once it is set to _true_, the controlled access of public folders is enabled.
```

Author: Made changes to config per review. But I feel the True and false part can be left as it is.

> 4. Enable **PublicFolderShowClientControl** parameter on the organization configuration. By default, the value of this parameter is also ‘false’ and once it is set to ‘true’, the controlled access of public folders is enabled.

Reviewer: ```suggestion
4. Enable the **PublicFolderShowClientControl** parameter on the organization configuration. By default, the value of this parameter is also ‘false’ and once it is set to ‘true’, the controlled access of public folders is enabled.
```
